src/views/product-root-docs-path-landing/components/marketing-content/marketing-content.module.css

```diff
@@ -4,6 +4,10 @@
 ***
 */
 
+.heading {
+  composes: g-offset-scroll-margin-top from global;
```

Comment:
**Nitpick ⛏️**: We have two separate values for accessing the navigation and alert banner offsets. Wonder if `--sticky-bars-height` can replace `--full-header-height`? Not sure about the loading order of the CSS.

Reply:
@alexcarpenter oh nice catch! 🙌🏻 Can you create a ticket to address this separately on this board? https://app.asana.com/0/1202097197789424/list